Pages/contact.html

```diff
@@ -3,8 +3,63 @@
 <head>
     <meta charset="UTF-8">
     <title>Meher Baba</title>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Acme Web Design</title>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
+          integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.css"/>
+    <link rel="stylesheet" href="../CSS/Style.css">
+
+
 </head>
 <body>
-<p> Do you have any questions/suggestions or something to say? Please feel free to write to us. We hear you. </p>
+
+<div class="container">
+    <h1 class="brand"><span>Dream Star</span> Pro </h1>
+    <div class="wrapper animated bounceInLeft">
+        <div class="company-info">
+            <h3>Dream Star Pro</h3>
+            <ul>
+                <li><i class="fa fa-road"></i> Germany</li>
+                <!--  <li><i class="fa fa-phone"></i> (555) 555-5555</li> -->
+                <li><i class="fa fa-envelope"></i> dreamstarpro@pm.me</li>
+            </ul>
+            <p> Do you have any questions/suggestions or something to say?</p>
+            <p> Please feel free to write to us. We hear you. </p>
```

Comment:
Repeated twice

Reply:
Which one is repeated?  :confused:

Reply:
" Do you have any questions/suggestions or something to say?
  Please feel free to write to us. We hear you."
The above is repeated twice in code and also when I checked in html format in web browser it was appearing two times